packages/server/lib/routes.js

```diff
@@ -59,6 +60,14 @@ module.exports = ({ app, config, getRemoteState, networkProxy, project, onError
     res.sendFile(file, { etag: false })
   })
 
+  app.get('/__bundled/*', (req, res) => {
```

Comment:
minor: can we namespace this to `/__cypress/bundled` so it's consistent with the other conventions?